srcpkgs/cawbird/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'cawbird'
+pkgname=cawbird
+version=1.2.0
+revision=1
+build_style=meson
+hostmakedepends="pkg-config vala glib-devel"
+makedepends="gtk+3-devel json-glib-devel sqlite-devel libsoup-devel gettext-devel vala-devel gst-plugins-base1-devel gst-plugins-bad1-devel gst-libav gspell-devel rest-devel"
+depends="rest gspell gst-plugins-bad1 gst-libav gstreamer1 gst-plugins-good1 sqlite libsoup gettext glib gtk+3 json-glib"
```

Comment:
Most of these are detected automatically, remove.

https://travis-ci.org/github/void-linux/void-packages/jobs/725749411#L878